CoreFoundation/Stream.subproj/CFSocketStream.c

```diff
@@ -110,7 +110,7 @@ static struct {
 
```

Comment:
@parkera so there is no way to get this functionality on Linux? 
 CFNetwork is no longer available as open source https://openradar.appspot.com/7195752